src/status_im2/contexts/onboarding/generating_keys/view.cljs

```diff
@@ -10,6 +10,12 @@
     [status-im2.contexts.onboarding.generating-keys.style :as style]
     [utils.i18n :as i18n]))
 
+(def first-title-display-time 2000)
+(def second-title-start-time 2500)
+(def second-title-display-time 1000)
+(def third-title-start-time 5100)
```

Comment:
@mohsen-ghafouri this looks good. As far as I recall there was nothing particularly special about these numbers other than we left time for each animation to run. @briansztamfater worked on this with the designers and we found these times to work. This looks good here 👍

Reply:
Thanks @J-Son89 for the clarification